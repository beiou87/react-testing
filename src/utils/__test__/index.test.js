import * as utils from '..';

describe('utils', () => {
  it('exports modules correctly', () => {
    expect(Object.keys(utils)).toMatchSnapshot();
  });
});
