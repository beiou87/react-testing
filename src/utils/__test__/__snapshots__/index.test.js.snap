// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils exports modules correctly 1`] = `
Array [
  "timer",
  "infiniteTimer",
]
`;
